content/en/Platform Deep Dive/Cobalt Account/account-recovery.md

```diff
@@ -11,6 +11,10 @@ description: >
 If you have problems signing in, refer to the instructions on this page. To get more help, reach out to {{% csm-support %}}.
 {{% /pageinfo %}}
 
+{{< alert title="Tip" color="primary" >}}
+Before proceeding, ask your Organization Owner if SAML-based single sign-on (SSO) is enabled for your organization. If yes, sign in from the identity provider system, not the Cobalt {{% sign-in %}} page. For more troubleshooting tips, see [Can’t Sign In Using SAML SSO](#cant-sign-in-using-saml-sso), below.
```

Comment:
Since this is now the first instance of "identity provider" on this page, should we move the `(IdP)` acronym up here (from new line 102)? (Approving, since IMO this is not a blocker.)